### ISSUE TITLE
issue with getCandlestickBars 

### ISSUE THREAD

**Issue by Najeisleem** [binance-exchange/binance-java-api#366]
I'm calling the API with start and end time but its return wrong data, I have test Binance API directly, and its working fine

**Comment by tonyradz** [comment]
yep, something is wrong with parsing long from value something like this "34.234234" and it is not long, probably some api changes has been made. I would suggest to make all field as Strings

**Comment by Najeisleem** [comment]
so I must call it directly from finance no throw this library, right?

**Comment by Najeisleem** [comment]
:D thanks alot man

**Comment by antanas-radzevicius-tgo** [comment]
No problem 😉

**Comment by Najeisleem** [comment]
can you please provide me with full example because still not worling
